crates/web-sys/webidls/Clipboard.webidl

```diff
@@ -46,6 +32,20 @@ dictionary ClipboardItemOptions {
   PresentationStyle presentationStyle = "unspecified";
 };
 
+typedef sequence<ClipboardItem> ClipboardItems;
+
+[SecureContext, Exposed=Window]
+interface Clipboard : EventTarget {
+  Promise<ClipboardItems> read(optional ClipboardUnsanitizedFormats formats = {});
+  Promise<DOMString> readText();
+  Promise<undefined> write(ClipboardItems data);
+  Promise<undefined> writeText(DOMString data);
+};
+
+dictionary ClipboardUnsanitizedFormats {
```

Comment:
I wasn't sure if I should remove this, because it's Chrome-only.

Reply:
We generally tend to follow the spec regardless of browser supprt.

Reply:
This is a bit of a special case, because this API was merged without consensus, but I don't want to relitigate this here :melting_face:

Reply:
Lets leave this part as unstable for sure then.